apps/src/p5lab/spritelab/defaultSprites.json

```diff
@@ -81,32 +17,6 @@
       "version": "wAQoTe9lNAp19q.JxOmT6hRtv1GceGwp",
       "categories": ["animals"]
     },
-    "44c5937d-c5c0-4676-bd0c-f7a86e99dd98": {
-      "name": "bee",
-      "sourceUrl": "https://studio.code.org/api/v1/animation-library/spritelab/b2QZ1J9ww5XYdjExrVb7lWgP2q6Gfx1C/category_animals/bee.png",
-      "frameSize": {
-        "x": 62,
-        "y": 50
-      },
-      "frameCount": 1,
-      "looping": true,
-      "frameDelay": 2,
-      "version": "b2QZ1J9ww5XYdjExrVb7lWgP2q6Gfx1C",
-      "categories": ["animals"]
-    },
-    "45b457bb-5b22-48f7-8664-4b38957d7799": {
-      "name": "brown bunny",
-      "sourceUrl": "https://studio.code.org/api/v1/animation-library/spritelab/uPIxfWA_cXFQ3rZjtIfd5beAWtSMrP1l/category_animals/bunny1_ready.png",
-      "frameSize": {
-        "x": 120,
-        "y": 191
-      },
-      "frameCount": 1,
-      "looping": true,
-      "frameDelay": 2,
-      "version": "uPIxfWA_cXFQ3rZjtIfd5beAWtSMrP1l",
-      "categories": ["animals"]
-    },
     "d15af884-fb27-455f-b73a-899d07ce8975": {
       "name": "purple bunny",
```

Comment:
Why do we want to remove all the other animations for Sprite Lab? I assume we still want all 40ish to show in new projects. 
Separately, in case it's relevant: Ff a Sprite Lab level is loaded without any animations from the backgrounds category, we usually load in all the default backgrounds so that the `set background` block still functions with choices.

Reply:
Since this is the back-up list of sprites that only loads when the call to S3 fails, we could make this as long or short as it is. My main goal here is to move this from being the reference list of default sprites to just an emergency back-up. I think cutting this list to a small number conceptually distinguishes it from the reference list on S3 and makes it clear that it is not the reference list. That being said, we could certainly keep, like 6, 10, 12, sprites on this list for a more robust fallback experience. Curious to hear your thoughts on that.

Reply:
Where do we load the backgrounds from?

Reply:
I'm not sure where we actually get the backgrounds from...
That plan sounds goods to me. Could we add the grid background so there's one of each type of animation available so no blocks appear broken?